### ISSUE TITLE
`SetPass` does not create `pass` entry `GoJira/<user>`

### ISSUE THREAD

**Issue by kBite** [Netflix-Skunkworks/go-jira#97]
As stated in PR #91, `go-jira` does not create `pass` entry `GoJira/<user>`. The first time using `go-jira` asks for my password, but won't store it.

```
$ jira fields
Jira Password [user]: ********
$ echo $?
0
$ pass show | grep -i GoJira; echo $?
1
```
Configration is correct as I can use `go-jira` with `pass` after manually adding my account to pass.

**Comment by coryb** [comment]
Hey, thanks for the updates.  I am not ignoring you, just don't have a lot of time to look into this.   I hope to be able to check into the issue this weekend.
-Cory

**Comment by coryb** [comment]
I think this might have been fixed in #104

**Comment by coryb** [comment]
Closing, reopen if still an issue.
